sdks/python/build.gradle

```diff
@@ -155,9 +155,24 @@ tasks.register("postCommitYamlIntegrationTests") {
   dependsOn ":sdks:java:io:google-cloud-platform:expansion-service:build"
 
   doLast {
+    def testSet = project.findProperty('yamlTestSet') ?: 'data'
```

Comment:
Rather than defaulting to this, I think we should still probably try running everything when this command is invoked (since this is expected to work locally).